setup.py

```diff
@@ -21,7 +21,7 @@
 # starting with packages common across all setups
 install_requires = ['requests', 'beautifulsoup4', 'lxml', 'netCDF4']
 # packages with Fortran code
-fortran_install = ['pysatCDF', 'madrigalWeb', 'h5py', 'PyForecastTools']
+fortran_install = ['pysatCDF', 'PyForecastTools']
```

Comment:
Is `model_utils` still a part of this branch?  If not, we can remove `PyForecastTools`.